docs/modules/ROOT/examples/operator/bundle.yaml

```
@@ -98,6 +95,109 @@ status:
   conditions: []
   storedVersions: []
 ---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
+  name: managementcenters.hazelcast.com
+spec:
+  group: hazelcast.com
+  names:
+    kind: ManagementCenter
+    listKind: ManagementCenterList
+    plural: managementcenters
+    singular: managementcenter
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: ManagementCenter is the Schema for the managementcenters API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: ManagementCenterSpec defines the desired state of ManagementCenter.
+            properties:
+              externalConnectivity:
+                description: Configuration to expose Management Center to outside.
+                properties:
+                  type:
+                    default: LoadBalancer
+                    description: 'Specifies how Management Center is exposed Valid values are: - "ClusterIP" - "NodePort" - "Loadbalancer" (default)'
```

```suggestion
                    description: 'How Management Center is exposed. Valid values are: - "ClusterIP" - "NodePort" - "Loadbalancer" (default)'
```

Do we need to give valid values here if we have the `enum` field?
Also, is there somewhere we can describe what each of these values mean?